### PR TITLE
Modified script:jaeger-client-go_rhel_8.3.sh

### PR DESCRIPTION
**Pull request by rajusah15**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@rajusah15 do we know why the travis check is failing?

**Comment by rajusah15** [comment]
> @rajusah15 do we know why the travis check is failing?

@seth-priya will check and will do the required changes.

**Comment by rajusah15** [comment]
trigger build.

File: g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh [hunk 1]
```diff
@@ -21,38 +24,38 @@ PACKAGE_URL=https://github.com/uber/jaeger-client-go
 
 yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
 
-wget https://golang.org/dl/go1.16.1.linux-ppc64le.tar.gz && tar -C /bin -xf go1.16.1.linux-ppc64le.tar.gz && mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
-
+wget https://golang.org/dl/go1.15.linux-ppc64le.tar.gz && tar -C /bin -xf go1.15.linux-ppc64le.tar.gz && mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
```
Comment (afsanjar) [inline g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh:27]: Is there a way to make GOLANG version dynamic?

File: g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh [hunk 2]
```diff
@@ -66,17 +69,19 @@ function test_with_master(){
...
 	echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
 	# Ensure go.mod file exists
-	go mod init
+	if [! -f go.mod ]; then
+	    go mod init
+	else
+	    go mod init $PACKAGE_NAME/$PACKAGE_VERSION
 	if ! go test ./...; then
 		test_with_master_without_flag_u
 		exit 0
```
Comment (afsanjar) [inline g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh:81]: Why exit 0 if there is a failure?

File: g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh [hunk 3]
```diff
@@ -109,16 +117,19 @@ if ! go get -d -u -t $PACKAGE_NAME@$PACKAGE_VERSION; then
...
 echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
 # Ensure go.mod file exists
-go mod init
+if [ ! -f go.mod ]; then
+    go mod init
+else
+    go mod init $PACKAGE_NAME/$PACKAGE_VERSION 
 if ! go test ./...; then
 	test_with_master
 	exit 0
```
Comment (afsanjar) [inline g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh:129]: Not sure what the action is in the case of failures.

File: g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh (hunk 1 above)
Comment (rajusah15) [reply g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh:27]: @afsanjar Yes

**Comment by rajusah15** [comment]
Will raise a Fresh PR as it has generated by automation and having some issue.

**Pull request closed without merging**